downstream/modules/aap-on-azure/proc-azure-configure-ad-sso.adoc

```diff
@@ -34,14 +34,14 @@ To register the {ControllerName} instance in Azure, you must supply the _Azure A
 . Ensure that you are using the tenant where you deployed {PlatformNameShort}.
 . Type `Azure Active Directory` in the search bar.
 . Select *Azure Active Directory* from the search results.
-. Under *Manage* in the menu options, click *App registrations*.
-. In the *App registrations* page, click *+ New registration*.
+. Under *Manage* in the menu options, click btn:[App registrations].
```

Comment:
I believe this is a menu cascade selection. I suggest:

From the menu options, select menu:Manage[App registrations].